### PR TITLE
Update __index.md

### PR DESCRIPTION
**Pull request by cottage14**
Minor text edits

### CODE REVIEW

File: source/calendars/__index.md [hunk 1]
```diff
@@ -7,39 +7,37 @@ This calendar lists major events relating to The Apache Software Foundation
 and our many Apache projects.  We also have an automated
 [listing of upcoming Apache-related Meetups][5] (smaller events) available.
 
-This Calendar is maintained by Apache Community Development volunteers (ComDev);
-to have your event listed considered for addition please mail [dev@community.apache.org][1].
-Note that to be listed in this calendar events must <b>already</b> conform
+Apache Community Development (ComDev) volunteers maintain this calendar. To have your event considered for inclusion, email details to [dev@community.apache.org][1].
+We list events that <b>already</b> conform
```
Comment (rlenferink) [inline source/calendars/__index.md:11]: Prefer to use markdown here

```suggestion
We list events that **already** conform
```

File: source/calendars/__index.md [hunk 2]
```diff
@@ -7,39 +7,37 @@ This calendar lists major events relating to The Apache Software Foundation
...
 
-In addition to major conferences that have applied for approval with us, there are a wide range of other
+In addition to major conferences that have applied for approval with us, there is a wide range of other
 events which feature Apache projects and technologies. A list of upcoming
 [Apache related Meetups is available][5], powered in part by [this script][6]
-(patches welcome!). Secondly, a crowd-sourced list of related events is maintained
+(patches welcome!). Second, a crowd-sourced list of related events is maintained
 as a [Lanyrd Guide][4], and we encourage people to help list
-Apache project related events there.
+Apache-project-related events there.
```
Comment (rlenferink) [inline source/calendars/__index.md:40]: Why the dashes? Is this some kind of convention?

Comment (rbowen) [reply source/calendars/__index.md:40]: It's a bit of a moot point - we should be removing any mention of Lanyrd, as that site is long-gone.

**Comment by ShaneCurcuru** [comment]
Sorry for the delay - but other changes like removing data about calendars from here mean this patch is moot now.

**Pull request closed without merging**